job-server/src/main/resources/application.conf

```diff
@@ -124,6 +129,26 @@ spark {
 akka {
   # Use SLF4J/logback for deployed environment logging
   loggers = ["akka.event.slf4j.Slf4jLogger"]
+
+  actor {
+    provider = "akka.cluster.ClusterActorRefProvider"
```

Comment:
Always spin up a cluster even for those not needed > 1 context? My one thought is it's a lot of chatter for not needing this new feature, unless leveraging things like leader election etc regardless :)